src/java/org/apache/cassandra/repair/autorepair/RepairTokenRangeSplitter.java

```diff
@@ -808,8 +680,6 @@ public String toString()
     protected static class SizedRepairAssignment extends RepairAssignment {
```

Comment:
Outside of description, this class is now the same as `RepairAssignment`, wonder if we could just promote 'description' up to `RepairAssignment` as I think it might be generally useful to annotate `RepairAssignment` with descriptions?